vizro-core/src/vizro/models/_components/container.py

```diff
@@ -14,6 +14,16 @@
 
 
 class Container(VizroBaseModel):
+    """Container component. [`Container`][vizro.models.Container].
+
+    Args:
+        type (Literal["container"]): Defaults to `"container"`.
+        components (List[ComponentType]): See [ComponentType][vizro.models.types.ComponentType]. At least one component
+            has to be provided.
+        title (str): Title to be displayed.
+        layout (Optional[Layout]): Layout to place components in. Defaults to `None`.
+    """
+
     type: Literal["container"] = "container"
     components: List[ComponentType]
     title: Optional[str]
```

Comment:
@antonymilne This means the user has to enter a title, but that title can be `None` right?

@nadijagraca Why is this no `Field` with a description, similar to page?

Reply:
The below line (which I cannot comment on) should be changed to `Layout = None` or did I misunderstand https://github.com/mckinsey/vizro/pull/210#

Reply:
Good catch. Changed it the `layout`.

Also, @huong-li-nguyen, @petar-qb title cannot be optional when `Container` is used within `Tabs`, because we don't have a default value and as a result of that users cannot access the content under that tab. Something to think about when you start to work on new `title` implementation.